tasks.py

```diff
@@ -1176,8 +1185,16 @@ def __schedule_json_to_dynamo(ctx, json_name):
 
     classes = json.load(file)
 
-    for item in classes.values():
-        table.put_item(Item=item)
+    try:
```

Comment:
This is fine too, but got me thinking it would be nice if we had a set of python lint rules that we re-used across projects (ala eslint). I would want it to complain about using `file = open(blah blah)` + `file.close()` vs a `with` statement that automatically closes the file handle.

Reply:
Yes, definitely.